{
  "Logging": {
    "LogLevel": {
      "Default": "Warning"
    }
  },
  "AllowedHosts": "*",
  "ConnectionStrings": {
    "AppConnString": "Data Source=(localdb)\\MSSQLLocalDB;Initial Catalog=CCFWorkflow;Integrated Security=True;Connect Timeout=30;Encrypt=False;TrustServerCertificate=False;ApplicationIntent=ReadWrite;MultiSubnetFailover=False"
  },
  "Auth0": {
    "Domain": "dev-7z2owzae.us.auth0.com",
    "ApiIdentifier": "4jytNlocTJIe59j6l4ZP879HJriUteJp"
  }
  "An error occurred while starting the application.
.NET Core 4.6.29017.01 X86 v4.0.0.0"
}
